package.json

```diff
@@ -8,6 +8,7 @@
         "start": "ng serve",
         "build": "ng build",
         "test": "ng test",
+        "style": "ng serve --sourcemap --extractCss",        
```

Comment:
This allows you to view the source maps of all the styles, very handy when debugging style issues....

Reply:
:+1::+1: